python/feather/ext.pyx

```diff
@@ -24,7 +24,7 @@ from cython.operator cimport dereference as deref
 from libfeather cimport *
 
 import pandas as pd
-import pandas.core.common as pdcom
+import pandas.api.types as pdapi
```

Comment:
Very good point and thanks a lot for the comment (I am pretty new here).

I forced pandas to 0.18.0 [https://travis-ci.org/guillett/feather/builds/166268471](url) and it failed as expected.

I added your suggestions to a branch on my repository [https://github.com/guillett/feather/tree/type_check_backward_compat](url) and run Travis for pandas 0.18.0 [https://travis-ci.org/guillett/feather/builds/166265966](url) and 0.19.0 [https://travis-ci.org/guillett/feather/builds/166265267](url).

Should I push those changes to the branch I used in my pull request?
Thanks again.